cli/test/org/partiql/cli/CliTestUtility.kt

```diff
@@ -26,6 +26,7 @@ fun makeCliAndGetResult(
     val cli = Cli(
         valueFactory,
         input?.byteInputStream(Charsets.UTF_8) ?: EmptyInputStream(),
+        InputFormat.ION,
```

Comment:
For consistency with the other `Cli` constructor args, I'd not hardcode the `inputFormat` argument but would rather pass it as an argument to `makeCliAndGetResult`.